packages/ui/cypress/e2e/login.cy.ts

```diff
@@ -0,0 +1,24 @@
+import { injectedAccounts } from '../fixtures/injectedAccounts'
+import { localHost } from '../fixtures/landingData'
+
+describe('login-accounts', () => {
+  beforeEach(() => {
+    cy.visit(localHost)
+  })
+
+  it('Connects with Alice', () => {
+    cy.initExtension(injectedAccounts)
+    cy.get('[data-cy="button-menu-connect"]').click()
```

Comment:
I'd move element `get` like this to a page object but we can tidy up later

Reply:
absolutely